src/data/set/basic.lean

```diff
@@ -1407,6 +1418,7 @@ end preimage
 /-! ### Image of a set under a function -/
 
 section image
+variables {f : α → β}
```

Comment:
```suggestion
```
Seems weird to introduce a variable if you're going to redeclare it on all the subsequent lemmas.

Reply:
I'm using this new variable twice, so I'll keep it.

Reply:
Well there are about 20 lemmas in this section that declare a new version of `f` rather than use this new one